src/Components/test/E2ETest/Tests/HttpClientTest.cs

```diff
@@ -1,43 +1,50 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
 using BasicTestApp.HttpClientTest;
-using Microsoft.AspNetCore.Components.E2ETest.Infrastructure;
 using Microsoft.AspNetCore.Components.E2ETest.Infrastructure.ServerFixtures;
 using Microsoft.AspNetCore.E2ETesting;
 using OpenQA.Selenium;
 using OpenQA.Selenium.Support.UI;
-using System;
-using System.Net.Http;
-using System.Threading.Tasks;
+using TestServer;
 using Xunit;
 using Xunit.Abstractions;
 
 namespace Microsoft.AspNetCore.Components.E2ETest.Tests
 {
-    public class HttpClientTest : BasicTestAppTestBase, IClassFixture<AspNetSiteServerFixture>
+    public class HttpClientTest : BrowserTestBase,
+        IClassFixture<DevHostServerFixture<BasicTestApp.Program>>, IClassFixture<BasicTestAppServerSiteFixture<CorsStartup>>
```

Comment:
Should we introduce another base-class here? Noticing that this doesn't use `ServerTestBase` and had to duplicate the log clearing logic.

Reply:
Switched it to use ServerTestBase